packages/eas-cli/src/utils/expoUpdatesCli.ts

```diff
@@ -26,8 +26,11 @@ export async function expoUpdatesCommandAsync(projectDir: string, args: string[]
   try {
     return (await spawnAsync(expoUpdatesCli, args)).stdout;
   } catch (e: any) {
-    if (e.stderr) {
-      if ((e.stderr as string).includes('Invalid command')) {
+    if (e.stderr && typeof e.stderr === 'string') {
+      if (
+        e.stderr.includes('Invalid command') || // SDK 51+
```

Comment:
ditto from the other PR https://github.com/expo/eas-build/pull/356#discussion_r1515004850